securedrop/crypto_util.py

```diff
@@ -215,15 +210,15 @@ def genkeypair(self, name, secret):
         ...     u'P'
         u'P'
 
-        :param str name: The source's filesystem id (their codename, salted
+        :param name: The source's filesystem id (their codename, salted
                          with SCRYPT_ID_PEPPER, and hashed with scrypt).
-        :param str secret: The source's codename.
+        :param secret: The source's codename.
         :returns: a :class:`GenKey <gnupg._parser.GenKey>` object, on which
                   the ``__str__()`` method may be called to return the
                   generated key's fingeprint.
 
         """
-        name = clean(name)
```

Comment:
clean() was always returning the same name.

Reply:
:+1: